backend/ctj_api/models.py

```diff
@@ -18,3 +18,68 @@ class Opportunities(models.Model):
 
     class Meta:
         db_table = "opportunities"
+
+
+class CommunityOfPractice(models.Model):
+
+    class PracticeAreas(models.TextChoices):
+        DATA_SCIENCE = 'data_science', 'Data Science'
+        ENGINEERING = 'engineering', 'Engineering'
+        OPS = 'ops', 'Ops'
+        PROJECT_MANAGEMENT = 'project_management', 'Project/Product Management'
+        UI_UX = 'ui_ux', 'UI/UX'
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    practice_area = models.CharField(max_length=50, choices=PracticeAreas.choices)
+    description = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = "community_of_practice"
+        verbose_name = "Community of Practice"
+        verbose_name_plural = "Communities of Practice"
+
+    def __str__(self):
+        return f"{self.get_practice_area_display()}"
```

Comment:
`self.get_practice_area_display`

What does this function do? I don't see its definition anywhere, unless I missed something 😕

Reply:
I believe this method is supposed to return the value set in the `choices`. I found the documentation [here](https://docs.djangoproject.com/en/5.1/ref/models/instances/#django.db.models.Model.get_FOO_display).